### ISSUE TITLE
Incorrect example

### ISSUE THREAD

**Issue by yinhx3** [OfficeDev/office-js-docs-pr#4870]
https://learn.microsoft.com/en-us/office/dev/add-ins/develop/dialog-api-in-office-add-ins#cross-domain-messaging-to-the-dialog-runtime
```
function onMessageFromParent(arg) {
    if (arg.origin === "https://addin.fabrikam.com") {
        // process message
    } else {
        dialog.close();
        showNotification("Messages from " + arg.origin + " aren't accepted.");
    }
}
```
`dialog.close();` can't run in a dialog. Should post message to the host to close a dialog corroding to the section below

**Comment by AlexJerabek** [comment]
Thanks for reporting this @yinhx3.

@samantharamon, could you please investigate?

**Comment by samantharamon** [comment]
Hi @yinhx3,

Thanks for catching this! We created a PR to fix it (see #4878). Once the change is merged, please note that this issue will be automatically closed. If you have additional feedback or require any assistance, please don't hesitate to reopen this issue or create a new one.
